Sources/TuistCore/Graph/Graph.swift

```diff
@@ -439,6 +439,14 @@ public class Graph: Encodable, Equatable {
             .filter { validProducts.contains($0.target.product) }
     }
 
+    public func appClipsDependency(path: AbsolutePath, name: String) -> TargetNode? {
+        guard let targetNode = findTargetNode(path: path, name: name) else {
+            return nil
+        }
+
+        return targetNode.targetDependencies.first { $0.target.product == .appClip }
```

Comment:
Question: can an app have multiple app clips? or is it restricted to one? if so, this could be another good lint rule.

Reply:
it appears to be restricted to one per app